### ISSUE TITLE
Selection indicator won't move

### ISSUE THREAD

**Issue by aruuin** [PageMenu/PageMenu#395]
Hello,
My issue is the selection indicator won't move and selected font won't change if I tap to next item. However the content changes if I tap from the menu, if I swipe the content to show next view controller, it only shows blank view.
It works fine on the simulator, but not on real device (tested on iOS 9 & 10).
Here's my code using PageMenu:
`for genre in genres! {
            let storyboard = UIStoryboard(name: "Home", bundle: nil)
            let vc = storyboard.instantiateViewController(withIdentifier: "MyViewController") as! MyViewController
            vc.title = genre.name!
            viewControllers.append(vc)
        }
        let parameters:[CAPSPageMenuOption] = [
            CAPSPageMenuOption.scrollMenuBackgroundColor(Colors.grey95),
            CAPSPageMenuOption.selectionIndicatorHeight(2),
            CAPSPageMenuOption.selectionIndicatorColor(Colors.fadedRed),
            CAPSPageMenuOption.selectedMenuItemLabelColor(Colors.black),
            CAPSPageMenuOption.unselectedMenuItemLabelColor(Colors.textGrey),
            CAPSPageMenuOption.menuItemFont(Fonts.nexaBold14!),
            CAPSPageMenuOption.menuItemSeparatorWidth(0),
            CAPSPageMenuOption.menuHeight(44),
            CAPSPageMenuOption.menuMargin(0),
            CAPSPageMenuOption.menuItemMargin(0),
            CAPSPageMenuOption.scrollAnimationDurationOnMenuItemTap(250),
        ]
        
        pageMenu = CAPSPageMenu(viewControllers: viewControllers, frame: CGRect(x:0,y:64,width:UIScreen.main.bounds.width,height:UIScreen.main.bounds.height-64), pageMenuOptions: parameters)
        pageMenu?.delegate = self
        
        self.view.addSubview(pageMenu!.view)`

**Comment by abhijith2smb** [comment]
Same issue in Swift 3.

**Comment by kirti301290** [comment]
Facing same issue 
pls provide solution if  solved

**Comment by Abdul-Hannan** [comment]
Need solution please

**Comment by nonesc** [comment]
+1

**Comment by kirti301290** [comment]
i found solution  removed pod file and put manually  file instead of pod it worked

**Comment by mdilaveroglu** [comment]
Solution on #112 fixed my problem.

**Comment by maksamster** [comment]
The link was repaired
https://github.com/orazz/PageMenu

**Comment by kcyuanGit** [comment]
I encountered the same issue.
It was solved by removing Pod and then manually added the CAPSMenu source files.

**Comment by Loviza** [comment]
https://github.com/PageMenu/PageMenu/issues/112
resolved my issue

**Comment by ElectroBuddha** [comment]
I had scrolling issues only on iPhone X, and I can confirm that #112 resolved my issue also.

**Comment by muratyasarr** [comment]
Just try cmd + left arrow and get simulator on landscape mode. And then turn back to portrait. Magic happened to me and it started working again :))

**Comment by anshumanburman** [comment]
@muratyasarr  great!!!!! it worked for me but this not a good solution. Now what i have done i just remove the pod and then added the manual files so now its working fine. This is xcode 10 issue pod is not working fine

**Comment by vasugryarasu** [comment]
@anshumanburman yes it's worked for me.. but when i use tableViewController as Child viewController tableview cells hiding at bottom.. i've custom navigation bar at top.. thanks

**Comment by MudassirAsghar22** [comment]
adding file manually worked for me as well, thanks everyone

**Comment by Collen-Roller** [comment]
Same issue, I was able to fix this using Swift 4.2 by adding the files locally.
